src/app/component/about/components/about-page/about-page.component.html

```diff
@@ -53,97 +54,87 @@ <h3 class="card-title-text">{{ 'about-us.steps.block-1.header' | translate }}</h
-          </div>
-          <div class="card-info">
-            <span class="main-card-text">
-              {{ 'about-us.steps.block-4.text' | translate }}
-            </span>
-            <a class="card-link disabled-link"
-               routerLink ="[]"
-               title="{{ 'about-us.disabled-link' | translate }}">
-              {{ 'about-us.steps.block-4.button' | translate }}
-            </a>
+        <div class="card-content card-story">
+          <div>
+            <div class="card-title">
+              <div class="card-title-number"></div>
+              <h3 class="card-title-text">{{ 'about-us.steps.block-4.header' | translate }}</h3>
+            </div>
+            <div class="card-info">
+              <span class="main-card-text">
+                {{ 'about-us.steps.block-4.text' | translate }}
+              </span>
+              <a class="card-link disabled-link" routerLink="[]" title="{{ 'about-us.disabled-link' | translate }}">
+                {{ 'about-us.steps.block-4.button' | translate }}
+              </a>
+            </div>
           </div>
+          <img src="assets/img/illustration-recycle.png" alt="illustration recycle">
         </div>
-        <img src="assets/img/illustration-recycle.png" alt="illustration recycle">
-      </div>
 
-      <div class="card-content card-people">
-        <img src="assets/img/illustration-people.png" alt="illustration people">
-        <div>
-          <div class="card-title">
-            <div class="card-title-number"></div>
-            <h3 class="card-title-text">{{ 'about-us.steps.block-5.header' | translate }}</h3>
-          </div>
-          <div class="card-info">
-            <span class="main-card-text" id="fifth-card-text">
-              {{ 'about-us.steps.block-5.text' | translate }}
-            </span>
-            <a class="disabled-link card-link"
-              title="{{ 'about-us.disabled-link' | translate }}"
-              routerLink ="[]">
-              {{ 'about-us.steps.block-5.button' | translate }}
-           </a>
+        <div class="card-content card-people">
+          <img src="assets/img/illustration-people.png" alt="illustration people">
+          <div>
+            <div class="card-title">
+              <div class="card-title-number"></div>
+              <h3 class="card-title-text">{{ 'about-us.steps.block-5.header' | translate }}</h3>
+            </div>
+            <div class="card-info">
+              <span class="main-card-text" id="fifth-card-text">
+                {{ 'about-us.steps.block-5.text' | translate }}
+              </span>
+              <a class="disabled-link card-link" title="{{ 'about-us.disabled-link' | translate }}" routerLink="[]">
+                {{ 'about-us.steps.block-5.button' | translate }}
+              </a>
+            </div>
           </div>
         </div>
       </div>
     </div>
   </div>
-</div>
+</main>
```

Comment:
Always leave one empty line at the end of file.

Reply:
got it